src/Dapr.Jobs/Models/DaprJobSchedule.cs

```diff
@@ -67,7 +67,9 @@ public static DaprJobSchedule FromCronExpression(CronExpressionBuilder builder)
     /// <returns></returns>
     public static DaprJobSchedule FromDateTime(DateTimeOffset scheduledTime)
     {
+#if NET6_0
         ArgumentNullException.ThrowIfNull(scheduledTime, nameof(scheduledTime));
```

Comment:
Can a `DateTimeOffset` (a `struct`) ever be null?

Reply:
Haha, bad habit.